code/datums/elements/snail_crawl.dm

```diff
@@ -23,8 +23,14 @@
 
 	if(snail.resting && !snail.buckled && lubricate(snail))
 		snail.add_movespeed_modifier(/datum/movespeed_modifier/snail_crawl)
+		ADD_TRAIT(snail, TRAIT_NEGATES_GRAVITY, TRAIT_GENERIC) //SKYRAT EDIT BEGIN - This is to prevent snails from achieving FTL speeds without gravity, think of it like snails affixing to walls irl.
+		snail.AddElement(/datum/element/forced_gravity, 0)
+		if(HAS_TRAIT(snail, TRAIT_SETTLER)) //This is to keep settlers from reaching FTL speeds too.
+			snail.remove_movespeed_modifier(/datum/movespeed_modifier/snail_crawl) //SKYRAT EDIT END
```

Comment:
```suggestion
		//SKYRAT EDIT BEGIN - This is to prevent snails from achieving FTL speeds without gravity, think of it like snails affixing to walls irl.
		ADD_TRAIT(snail, TRAIT_NEGATES_GRAVITY, TRAIT_GENERIC)
		snail.AddElement(/datum/element/forced_gravity, 0)
		if(HAS_TRAIT(snail, TRAIT_SETTLER)) //This is to keep settlers from reaching FTL speeds too.
			snail.remove_movespeed_modifier(/datum/movespeed_modifier/snail_crawl)
		//SKYRAT EDIT END
```